src/snowflake/snowpark/modin/plugin/extensions/index.py

```diff
@@ -708,8 +708,7 @@ def T(self, *args: Any, **kwargs: Any) -> Index:
         """
         return self
 
-    @index_not_implemented()
-    def all(self) -> None:
+    def all(self, *args, **kwargs) -> bool | ExtensionArray:
```

Comment:
can you add one sentence in the docstring below saying `*args, **kwargs` are added for compatibility but not used in Snowpark.